backend/sites/src/app/services/site/site.service.ts

```diff
@@ -1719,6 +1719,21 @@ export class SiteService {
       }
 
       this.sitesLogger.log('SiteService.processSRBulkUpdates() end');
```

Comment:
i think this logger should be just before 'return true'

Reply:
Done